src/components/site/Accordion.md

```diff
@@ -0,0 +1,170 @@
+```jsx
+const collapseItems = [
+  {
+    id: 1,
+    header: "Accordion Item #1",
+    subheader: "This is a subheader",
+    text: "Text goes here"
+  },
+  {
+    id: 2,
+    header: "Accordion Item #2",
+    subheader: "",
+    text: "Text goes here"
+  },
+  {
+    id: 3,
+    header: "Accordion Item #3",
+    subheader: "",
+    text: (
+      <img
+        width="300"
+        height="200"
+        alt="Stuff goes here"
+        src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
+        border="0"
+        vspace="0"
+        hspace="0"
+      />
+    )
+  },
+  {
+    id: 4,
+    header: "Accordion Item #4",
+    subheader: "",
+    text: <a href="https://www.w3schools.com">This is a link </a>
+  }
+];
+
+<Accordion
+  className="dg_section"
+  collapseItems={collapseItems}
+  toggleMultiOpen="dg_allowmultipleopen"
+/>;
+```
+
+### Accordion Dark
+
+```jsx
+const collapseItems = [
+  {
+    id: 11,
+    header: "Accordion Item #1",
+    subheader: "This is a subheader",
+    text: "Text goes here"
+  },
+  {
+    id: 12,
+    header: "Accordion Item #2",
+    subheader: "",
+    text: "Text goes here"
+  },
+  {
+    id: 13,
+    header: "Accordion Item #3",
+    subheader: "",
+    text: (
+      <img
+        width="300"
+        height="200"
+        alt="Stuff goes here"
+        src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
+        border="0"
+        vspace="0"
+        hspace="0"
+      />
+    )
+  },
+  {
+    id: 14,
+    header: "Accordion Item #4",
+    subheader: "",
+    text: <a href="https://www.w3schools.com">This is a link </a>
+  }
+];
+
+<Accordion className="dg_section dark" collapseItems={collapseItems} />;
```

Comment:
This needs to change slightly because of how web services will use this component. Web service will dump this in a section. That section may be white, dark or light. The accordion styles should change based on that section. So in this case your example would like.

```jsx
<Section className="dark">
  <Accordion collapseItems={collapseItems} />
</Section>
```

This will require you to change your css a bit.

Reply:
if you have questions about this, let me know.

Reply:
this should be done